portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase

```diff
@@ -1565,18 +1565,30 @@ definition {
 	@description = "Sign into portal by executing a SP initiated SSO login. This is done by clicking on the 'Sign In' link in the dockbar on the SP. After the SP is logged in, and if both instances has the same user and email address, the IdP will also be logged in."
 	@priority = "4"
 	test ViewSPInitiatedSSO {
+		property portal.upstream = "gabor";
 		property test.name.skip.portal.instance = "SAML#ViewSPInitiatedSSO";
 
+		takeScreenshot();
+
 		SAML.configureLiferaySAMLAsIdP(samlEntityID = "samlidp");
 
+		takeScreenshot();
+
 		SAML.configureLiferaySAMLAsSP(
 			samlEntityID = "samlsp",
-			spURL = "http://www.able.com:9080/");
```

Comment:
Hey @gaborlovasdotcom 
Additional to configureLiferaySAMLAsIdP, can you also try to remove every enclosing '/' in SAML.testcase?
I noticed there is double slash issue during my investigation.

Reply:
Yes I remove them, thanks for mentioning!